### DISCUSSION TITLE
Collaborators wanted

### DISCUSSION THREAD

**Discussion by pwaller** [pwaller/pyfiglet#99] [Announcements]
Hi! I'm going to move this to github.com/pyfiglet so that I don't represent a single point of failure.

@cjones @stefanor @LyesSaadi @peterbrittain would you be happy to join the org? The main thing would be to ensure that the project continues to have someone with access to it should something happen to me. The rate of contributions and amount of maintenance required is very low at this point.

Anyone else is welcome to join as well so long as I can establish some trust that you would be a good custodian of the project. I'm not looking for large amount of contributions, the intent is to keep this project stable and maintained. It helps if you have been involved in the past, and have public evidence that you could perform this role.

Thanks!

**Comment by LyesSaadi** [comment]
I wouldn't be opposed to it if no one else can. Since I'll be maintaining this in Fedora, if any bug is reported on my side, I guess it could have help having some access?

**Comment by peterbrittain** [comment]
Thanks for thinking of me.  I have to confess I don't have much spare time these days, so couldn't be a primary contact, but am happy to be a backup.

**Comment by Siddhesh-Agarwal** [comment]
I am interested to join. I have even made an API to use Pyfiglet in other projects. [Repo](https://github.com/Siddhesh-Agarwal/Figlet-API)

**Comment by stefanor** [comment]
Sure

**Comment by penymarkos** [comment]
Hello, yes, I can participate

در تاریخ پنجشنبه ۱۹ ژانویهٔ ۲۰۲۳،‏ ۱۵:۰۶ Stefano Rivera <
***@***.***> نوشت:
